internal/spdxlicense/generate/generate_license_list.go

```diff
@@ -91,3 +92,81 @@ func main() {
 		log.Fatalf("unable to generate template: %+v", err)
 	}
 }
+
+func processSPDXLicense(result LicenseList) map[string]string {
+	// first pass build map
```

Comment:
probably worth beefing up the comment since the two-pass approach is very intentional

Reply:
comment added